release.csx

```diff
@@ -0,0 +1,84 @@
+#r ".\packages\Octokit.0.26.0\lib\net45\Octokit.dll"
+#r ".\packages\system.runtime.interopservices.runtimeinformation\4.0.0\lib\net45\System.Runtime.InteropServices.RuntimeInformation.dll"
+
+using Octokit;
+
+const string repoOwner = "FakeItEasy";
+const string repoName = "FakeItEasy";
+const string remote = "upstream";
+
+var version = Args.FirstOrDefault();
+if (version == null)
+{
+    throw new Exception("No version supplied");
+}
+
+var releaseBranchName = $"release/{version}";
+
+var token = ReadTokenFromNetrc();
+
+var gitHubClient = new GitHubClient(new ProductHeaderValue("FakeItEasy-build-scripts"));
+gitHubClient.Credentials = new Credentials(token);
+Cmd("git", $"checkout -b {releaseBranchName}");
+Cmd("git", $"push {remote} {releaseBranchName}");
+
+Console.WriteLine($"Creating pull request to merge {releaseBranchName} into master...");
+var pr = await gitHubClient.PullRequest.Create(
+    repoOwner,
+    repoName,
+    new NewPullRequest($"Release {version}", releaseBranchName, "master"));
+Console.WriteLine($"Created pull request {pr.Number}: '{pr.Title}'");
+
+
+public static string ReadTokenFromNetrc()
+{
+    const string githubApiHost = "api.github.com";
+    var netrcContents = File.ReadAllText(Path.Combine(Environment.GetEnvironmentVariable("HOME"), ".netrc"));
+    var tokens = netrcContents.Split(new[] { ' ', '\t', '\r', '\n' }, StringSplitOptions.RemoveEmptyEntries);
+
+    int lastTokenToCheck = tokens.Length - 1;
+    int i = 0;
+    for (; i < lastTokenToCheck; i += 2)
+    {
+        if (tokens[i] == "machine" && tokens[i + 1] == githubApiHost)
+        {
+            break;
+        }
+    }
+
+    for (i += 2; i < lastTokenToCheck; i += 2)
+    {
+        if (tokens[i] == "password")
+        {
+            return tokens[i + 1];
+        }
+
+        if (tokens[i] == "machine" || tokens[i] == "default")
+        {
+            break;
+        }
+    }
+
+    throw new Exception($"Can't find password for {githubApiHost} in .netrc file");
+}
+
+public static void Cmd(string fileName, string args)

```

Comment:
Nearly identical to the version in {{build.csx}}. We could extract and consolidate, with the same "where to put the common stuff?" question from above.

Reply:
Yes, but I think it can wait. No need to worry about that in this PR.